mms/metrics/process_memory_metric.py

```diff
@@ -45,4 +47,6 @@ def check_process_mem_usage(stdin):
     for process in process_list:
         if not process:
             continue
-        logging.info("%s:%d", process, get_cpu_usage(process))
+        mem_utilization = get_cpu_usage(process)
+        if mem_utilization != 0:
+            logging.info("%s:%d", process, mem_utilization)
```

Comment:
Else, should we throw any errors as above?

Reply:
this metrics process is not thowing anything, there's no catching on java side. actually this logging.info is fine, it is seen by java and propagated to the logs